src/aiida_quantumespresso/workflows/functions/get_xspectra_structures.py

```diff
@@ -0,0 +1,306 @@
+# -*- coding: utf-8 -*-
+"""CalcFunction to analyse the symmetry properties of a crystal structure and its atomic sites.
+
+Returns a supercell with a marked absorbing atom for each symmetrically non-equivalent site in the system.
+"""
+from aiida import orm
+from aiida.common import ValidationError
+from aiida.common.constants import elements
+from aiida.engine import calcfunction
+from aiida.orm.nodes.data.structure import Kind, Site, StructureData
+from aiida.tools import spglib_tuple_to_structure, structure_to_spglib_tuple
+import numpy as np
+import spglib
+
+
+@calcfunction
+def get_xspectra_structures(structure, **kwargs):  # pylint: disable=too-many-statements
+    """Read a StructureData object using spglib for its symmetry data and return structures for XSpectra calculations.
+
+    Takes an incoming StructureData node and prepares structures suitable for calculation with
+    xspectra.x. The basic criteria for a suitable structure is for the cell parameters to be
+    sufficiently large to reduce spurious interactions between localised core-holes in
+    neighbouring periodic images and for one atomic site to be marked as the absorbing atom
+    site.
+
+    The default setting for the cell size is 8.0 angstrom, which can be changed by including
+    'supercell_min_parameter' as a Float node in the keyword arguments. Accepted keyword
+    arguments are:
+        - abs_atom_marker: a Str node defining the name of the absorbing atom Kind. The
```

Comment:
Here is the line with the error reported from the docs check.
```suggestion
    arguments are:

        - abs_atom_marker: a Str node defining the name of the absorbing atom Kind. The
```